charts/nri-metadata-injection/templates/admission-webhooks/job-patch/clusterrolebinding.yaml

```diff
@@ -15,6 +15,6 @@ roleRef:
   name: {{ template "nri-metadata-injection.fullname" . }}-admission
 subjects:
   - kind: ServiceAccount
-    name: {{ template "nri-metadata-injection.fullname" . }}-admission
+    name: {{ template "nri-metadata-injection.fullname" . }}
```

Comment:
does make sense to change this if deprecating it?

Reply:
it is to avoid the issue with the 64 char names in the ci